.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Build
         run: |
           ./scripts/download-openpnp-capture.sh v0.0.22
-          mvn -B clean package
```

Comment:
Removing clean is okay, but why remove -B?

Reply:
I just... liked the colors? :-S:

>  -B,--batch-mode                        Run in non-interactive (batch)
>                                        mode (disables output color)

ASCII term colors might have been problematic in the past in some terminals, but both GHA and modern terminals take it well nowadays... happy to put it back, no worries.